MinecraftClient/Protocol/ProtocolHandler.cs

```diff
@@ -767,7 +827,7 @@ public static string GetRealmsWorldServerAddress(string worldId, string username
         /// <param name="cookies">Cookies for making the request</param>
         /// <param name="result">Request result</param>
         /// <returns>HTTP Status code</returns>
-        private static int DoHTTPSGet(string host, string endpoint, string cookies, ref string result)
+        public static int DoHTTPSGet(string host, string endpoint, string cookies, ref string result)

```

Comment:
```suggestion
        private static int DoHTTPSGet(string host, string endpoint, string cookies, ref string result)
```
No need to switch this method to `public` except maybe for testing.

Reply:
Ahh yes, I switched this to public because I needed to use it in another class, but I found a smarter way to do it afterwards